crates/test_utils/src/agent.rs

```diff
@@ -144,7 +156,7 @@ impl AgentBuilder {
         });
         let is_tombstone = self.is_tombstone.unwrap_or(false);
         let url = self.url.unwrap_or(None);
-        let storage_arc = self.storage_arc.unwrap_or(DhtArc::FULL);
+        let storage_arc = self.storage_arc.unwrap_or_default();
```

Comment:
I want to be able to set this in tests and the default is `Empty`. Didn't break any existing tests so I'm assuming this value wasn't too important to anyone else yet